File: src/main/java/com/example/moexbondservice/service/BondService.java
package com.example.moexbondservice.service;

import com.example.moexbondservice.dto.StockPricesDto;
import com.example.moexbondservice.dto.StocksDto;
import com.example.moexbondservice.dto.TickersDto;

public interface BondService {
    StocksDto getBondsFromMoex(TickersDto tickers);

    StockPricesDto getPricesByFigies(TickersDto figies);
}

File: src/main/java/com/example/moexbondservice/service/BondService.kt
package com.example.moexbondservice.service

import com.example.moexbondservice.dto.StockPricesDto
import com.example.moexbondservice.dto.StocksDto
import com.example.moexbondservice.dto.TickersDto

interface BondService {
    fun getBondsFromMoex(tickers: TickersDto): StocksDto
    fun getPricesByTickers(figies: TickersDto?): StockPricesDto
}
